tasks.py

```diff
@@ -88,6 +89,22 @@ def test(cov=True, verbose=False, debug=False):
 
 
 @task
+def test_harvesters(cov=True, verbose=False, debug=False):
+    """
+    Runs all harvester tests in the 'tests/' directory
+    """
+    cmd = 'py.test tests'
+    args = ''
+    if verbose:
+        args += ' -v'
+    if debug:
+        args += ' -s'
+    if cov:
+        args += ' --cov-report term-missing --cov-config .coveragerc --cov scrapi'
+    map(lambda x: run('py.test tests/' + x + args) if('harvester' in x) else(), os.listdir('tests'))
```

Comment:
As an additional, stylistic note, I think maps are usually best used in functions where you don't have side-effects, but have a transformation that you want to perform on each element of the list. For example 
```
map(str, [1, 2, 3, 4])
```
Takes a list of integers and returns a list of strings. As another example, say you wanted only the first character of a list of strings, you could do:
```
map(lambda x: x[:1], ['hector', 'earl', 'lenny', 'larry', 'oswald'])
```
But when you have side-effects like the above function, and the return value of the function you are mapping isn't important to you, a for loop is probably the better choice.